packages/playground/src/weblets/profile_manager.vue

```diff
@@ -21,7 +21,8 @@
               Balance: <strong :style="{ color: '#76e2c8' }">{{ normalizeBalance(balance.free, true) }} TFT</strong>
             </p>
             <p>
-              Locked: <strong :style="{ color: '#76e2c8' }">{{ normalizeBalance(balance.locked, true) }} TFT</strong>
+              Locked:
+              <strong :style="{ color: '#76e2c8' }">{{ normalizeBalance(balance.locked, true) || 0 }} TFT</strong>
```

Comment:
Is it possible to do the same with the balance instead of a boolean value?

Reply:
The free balance will always be a number and even if not found, it'll return 0. However, this is not the case in free balance which returns undefined in case of not found.

![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/40770501/4a2cf0de-11fd-4676-a9e6-4ad59fded613)